src/foundry/client/pixi/board.d.ts

```diff
@@ -374,28 +374,30 @@ declare global {
      * Compute the blur parameters according to grid size and performance mode.
      * @param options - Blur options.
      */
-    protected _initializeBlur(options?: {
-      enabled: boolean;
+    protected _initializeBlur(
+      options?: InexactPartial<{
```

Comment:
I see that these properties were required before. Did I mark this as needing `InexactPartial`? If so I may have made a mistake.
Are these properties optional is what I'm asking.

Reply:
They are optional; double checked that the default was `{}`